wiki/dfg_nodes.md

```diff
@@ -0,0 +1,53 @@
+# Understanding Data Flow Graph (DFG) Node Labels in Flowr
+
+## Overview
+The Data Flow Graph (DFG) in Flowr represents the dependencies between different code components. Each node in the DFG contains specific labels that provide information about the operation being performed, the code span, and its relationship with other nodes.
```

Comment:
We usually use `flowR` as the name of the framework and only capitalize it if used as a subject at the start of a sentence.